### ISSUE TITLE
Missing links from the Understanding documents to the WCAG

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2954]
Since the new design the links from the Understanding documents back to the [WCAG page](https://www.w3.org/TR/WCAG21/) are missing

**Comment by mitchellevan** [comment]
The problem is no longer occurring, as described here: https://github.com/w3c/wcag/issues/3159#issuecomment-3184818347
